magit-filenotify.el

```diff
@@ -56,9 +56,46 @@
              collect (expand-file-name dir))
     :test #'string=)))
 
-(defvar magit-filenotify-data (make-hash-table)
+;; Use #'equal as test because watch-descriptors aren't always integers.  With
+;; inotify, descriptors can be lists like (17).
+(defvar magit-filenotify-data (make-hash-table :test #'equal)
   "A hash table to map watch-descriptors to a list (DIRECTORY STATUS-BUFFER).")
 
+(defvar magit-filenotify--idle-timer nil
+  "Timer which will refresh buffers when emacs becomes idle.")
+
+(defvar magit-filenotify-idle-delay 1.57
```

Comment:
Done in https://github.com/tsdh/magit-filenotify/commit/90e5c7798cea93e680799e2eec105d3dd6828ed0 now.